### PR TITLE
[New] Add zrender w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #9229
Related issue(s): #9726 #11170

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ecomfe/zrender
 * Official website (optional, not the repository): https://ecomfe.github.io/zrender/
 * NPM package url (optional): https://www.npmjs.com/package/zrender
 * License and its reference: [BSD-3-Clause](https://github.com/ecomfe/zrender#license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 212
   - Count of stars: 1989
   - Count of forks: 661
 * NPM download stats (optional):
   - Downloads in the last day: 301
   - Downloads in the last week: 8308
   - Downloads in the last month: 29,428

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@dakshshah96 @cdnjs/intern3 
Please help me review this PR, Thank you.

File: ajax/libs/zrender/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "zrender",
+  "version": "3.5.2",
+  "filename": "zrender.min.js",
+  "description": "A lightweight canvas library which providing 2d draw for ECharts.",
+  "keywords": [
+    "canvas",
+    "2d"
```
Comment (dakshshah96) [inline ajax/libs/zrender/package.json]: A couple of relevant keywords such as these can be added:
* draw
* echarts

**Comment by sufuf3** [comment]
@dakshshah96 I added two keywords.
Please help me review this PR, Thank you.

File: ajax/libs/zrender/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    {
+      "name": "Shuang Su",
+      "url": "https://github.com/100pah"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ecomfe/zrender.git"
+  },
+  "license": "BSD-2-Clause",
```
Comment (extend1994) [inline ajax/libs/zrender/package.json]: This should be `BSD-3-Clause`.
Ref: https://libraries.io/bower/zrender and https://libraries.io/npm/zrender

**Comment by sufuf3** [comment]
@extend1994 I modified it 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @dakshshah96 
Please help me review this PR, Thank you.

**Comment by pvnr0082t** [comment]
Just help rebase it onto the latest master branch. 
ping @hare1039 @qwe19272375 @dakshshah96

**Comment by sufuf3** [comment]
@pvnr0082t @maruilian11 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@hare1039 @qwe19272375 @pvnr0082t @maruilian11
Please help me review this PR, Thank you.

**Comment by maruilian11** [comment]
Please update the license info in the init comment.
`Manually add v1.0.0~v1.0.2 of zrender`  ---> `Manually add v1.0.0 ~ v1.0.2 of zrender`

**Comment by sufuf3** [comment]
@maruilian11 I updated the commit message and the license info in the init comment.
Please help me review this PR, thank you.

**Comment by maruilian11** [comment]
https://github.com/cdnjs/cdnjs/pull/11457#pullrequestreview-56000942

**Comment by sufuf3** [comment]
@maruilian11 I added v1.0.3 ~ v 2.1.1
Please help me review this PR, Thank you.

**Comment by maruilian11** [comment]
`v1.0.3 ~ v 2.1.1`?

**Comment by sufuf3** [comment]
@maruilian11 I updated the commit message 
place help me review this PR, thank you.

**Comment by sufuf3** [comment]
@maruilian11 Is this PR OK?
Thank you.

**Comment by sufuf3** [comment]
@cdnjs/intern3 @maruilian11 @pvnr0082t 
Please help me review this PR, Thank you.

File: ajax/libs/zrender/2.1.1/zrender.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2006 Google Inc.
```
Comment (hare1039) [inline ajax/libs/zrender/2.1.1/zrender.js]: This file contain nothing but comments. Do we need to include this file?

Comment (hare1039) [reply ajax/libs/zrender/2.1.1/zrender.js]: P.S. `zrender.js` seems empty from 1.0.3 ~ 2.1.1

Comment (sufuf3) [reply ajax/libs/zrender/2.1.1/zrender.js]: Please take a look line 29
This file can work by these codes.
https://github.com/ecomfe/zrender/blob/2.1.1/build/zrender.js#L29

Comment (sufuf3) [reply ajax/libs/zrender/2.1.1/zrender.js]: @hare1039 Please take a look line 29
This file can work by these codes.
https://github.com/ecomfe/zrender/blob/2.1.1/build/zrender.js#L29

**Comment by sufuf3** [comment]
@hare1039 `zrender.js` is minified from `zrender-original.js`...

**Comment by PeterDaveHello** [comment]
There is no need to manually add v1.0.3~v2 as I'll manually add them.

**Pull request merged**